### PR TITLE
Disable game mode voting over a certain cap

### PR DESCRIPTION
**Pull request by TiviPlus**
## Why It's Good For The Game

Yuhhh

## Changelog
:cl:
admin: Disabled game mode voting over a certain cap, admins may bypass
config: Config for said cap
/:cl:

### CODE REVIEW

File: config/config.txt [hunk 1]
```diff
@@ -190,3 +190,6 @@ GUEST_BAN
 
 ## Comment this out to disable checking for the cid randomizer dll. (disabled if database isn't enabled or connected)
 CHECK_RANDOMIZER
+
+##Set this number to however max players you want after which game mode votes cannot be started
```
Comment (Hughgent) [inline config/config.txt:194]:
```suggestion
##Set this number to the maximum number of players you want, after which game mode votes cannot be started
```

**Review by Hughgent — APPROVED** [review]
YOLO

**Comment by Hughgent** [comment]
ah thing to check.

If it disabled game mode vote, will it just be 'stuck' on crash? (or whatever gamemode is currently set)

**Comment by Wayland-Smithy** [comment]
Maybe someday...
image

**Comment by TiviPlus** [comment]
@Hughgent

**Comment by TiviPlus** [comment]
@Hughgent Admins keep having to cancel votes this PR is still waiting

**Comment by Hughgent** [comment]
Questions still unanswered

**Comment by TiviPlus** [comment]
@Hughgent wayland already posted the answer ages ago

**Comment by Hughgent** [comment]
No they didn't, because it's not here in this PR.

**Comment by SilencedMP5A5** [comment]
merge this already pleaseeeee

**Comment by TiviPlus** [comment]
@Hughgent Because this is a bandaid for the glaring problem that is highpop crash votes
Admins as well as I are fed up and by god I will selfmerge if I must

**Comment by Wayland-Smithy** [comment]
> Admins fed up. Bandaid. 

Pretty serious bandaid if you built in the future proofing with a new config option.
Can not the fed up admins apply a bandaid of their own and manually disable calling votes by all non-admins with their godlike powers?

**Comment by TiviPlus** [comment]
I go through on my threats

**Pull request merged**